models/semantic_segmentation/DUC/cityscapes_labels.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+#
+# Cityscapes labels
```

Comment:
Usage info? What about adding some usage info to the README?

Reply:
done!